gocourse07/birdaviary/internal/centralsystem/centralsystem.go

```diff
@@ -0,0 +1,32 @@
+package centralsystem
+
+import (
+	"birdaviary/internal/logger"
+	"birdaviary/internal/storage"
+	"sync"
+)
+
+type CentralSystem struct {
+	Memory *storage.Storage
```

Comment:
Зробімо всі поля приватними:
```suggestion
	memory *storage.Storage
```